UITest Templates/UITestProtocols.xctemplate/AccessibilityIdentifiable.swift

```diff
@@ -0,0 +1,36 @@
+//
+//  ___FILENAME___
+//  ___PROJECTNAME___
+//
+//  Created by ___FULLUSERNAME___ on ___DATE___.
+//  ___COPYRIGHT___
+//
+
+//
+// Add this file to Main Target as well
+//
+
+import UIKit
+
+protocol AccessibilityIdentifiable: AnyObject {
+
+    associatedtype Identifier: IdentifierProvidable
+}
+
+extension AccessibilityIdentifiable {
+
+    func setAccessibilityIdentifier(of view: UIAccessibilityIdentification?, with identifier: Identifier) {
+        view?.accessibilityIdentifier = identifier.identifier
+    }
+
+    func setAccessibilityIdentifier(with dictionary: [UIView?: Identifier]) {
+        dictionary.forEach(setAccessibilityIdentifier)
+    }
+}
+
+extension AccessibilityIdentifiable where Self: UIViewController {
+
+    func setAccessibilityIdentifierDomainToView() {
+        view.accessibilityIdentifier = Identifier.screenName
+    }
+}
```

Comment:
I think we should put these codes into the project as default, don't need to put them into `xctemplate`

Reply:
We only need to create files for the test module: `___VARIABLE_moduleName___FlowSpec.swift`, `___VARIABLE_moduleName___Identifier.swift`,  `___VARIABLE_moduleName___Screen.swift `

Reply:
My inten

Reply:
@blyscuit I think we can create a framework, or put these files as default into the project